tests/data/complex_poincare_disk_data.py

```diff
@@ -91,45 +91,22 @@ class ComplexPoincareDiskMetricTestData(_ComplexRiemannianMetricTestData):
     def inner_product_test_data(self):
         smoke_data = [
             dict(
-                scale=0.5,
-                tangent_vec_a=[[1j]],
-                tangent_vec_b=[[1j]],
-                base_point=[[0j]],
-                expected=[1 / 4],
-            ),
-            dict(
-                scale=1,
                 tangent_vec_a=[[3.0 + 4j]],
                 tangent_vec_b=[[3.0 + 4j]],
                 base_point=[[0.0]],
                 expected=[25],
             ),
-            dict(
-                scale=2,
-                tangent_vec_a=[[1j], [1j], [1j]],
-                tangent_vec_b=[[1j], [2j], [3j]],
-                base_point=[[0j], [0j], [0j]],
-                expected=[4, 8, 12],
-            ),
```

Comment:
keep data without scale and updating expected?

Reply:
As above.